src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -11,6 +11,20 @@
 using System.Threading;
 using Microsoft.Win32;
 
+#if CORERT
```

Comment:
@jkotas I think the guidelines aren't 100% accurate as they state that Interop files should be placed under Common\Interop but in coreclr/corert they are placed under src/Interop and shared/Interop.